app/views/devise/passwords/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "Forgot password" -%>
```

Comment:
Right, it pretty long how about just `Password reset`? I'd like to have the title match the action of the page.
We might also need to trim `Resend unlock instructions`.